core/arch/arm/include/sm/psci.h

```diff
@@ -23,8 +24,11 @@
 #define PSCI_PSCI_SET_SUSPEND_MODE	U(0x8400000f)
 #define PSCI_FN_STAT_RESIDENCY		U(0x84000010)
 #define PSCI_FN_STAT_COUNT		U(0x84000011)
+#define PSCI_SYSTEM_RESET2		U(0x84000012)
+#define PSCI_MEM_PROTECT		U(0x84000013)
+#define PSCI_MEM_CHK_RANGE		U(0x84000014)
```

Comment:
I'd prefer to just stick the `PSCI_` prefix to the name in the spec, that is, `PSCI_MEM_PROTECT_CHECK_RANGE`

Reply:
fixed